week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,13 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+
+var students = 104;
+var mentors = 56;
+var total = students + mentors;
+var percentageOfMentors = Math.round((mentors*100)/total);
+var percentageOfStudents = Math.round((students*100)/total);
```

Comment:
If you move these 5 variables below the function description, does it still work? Can you think of a way to use function parameters?

If you want some additional reading on parameters, this one might help: https://codeburst.io/parameters-arguments-in-javascript-eb1d8bd0ef04